### PR TITLE
Update getting_started_table.py

### PR DESCRIPTION
**Pull request by sabas**
Fix error reported by chrome console, React table expects the rows wrapped in thead or tbody

Post-merge checklist:

The master branch is auto-deployed to `dash.plot.ly`.
Once you have merged your PR, wait 5-10 minutes and check dash.plot.ly
to verify that your changes have been made.

- [x] I understand

If this PR documents a new feature of Dash:

- [ ] Comment on the original Dash issue with a link to the new docs.
- [ ] Reply to any community thread(s) asking for this feature.

If this PR includes a new dataset available at a remote URL:
- [ ] I have added this dataset to the `datasets/` folder
- [ ] I have added a mapping between the remote URL and the filename in the
`datasets/` folder into the `find_and_replace` dict in `dash_docs/tools.py`

### CODE REVIEW

File: dash_docs/tutorial/examples/getting_started_table.py [hunk 1]
```diff
@@ -9,12 +9,16 @@
 def generate_table(dataframe, max_rows=10):
     return html.Table(
         # Header
```
Comment (alexcjohnson) [inline dash_docs/tutorial/examples/getting_started_table.py]:
```suggestion
```

File: dash_docs/tutorial/examples/getting_started_table.py [hunk 2]
```diff
@@ -9,12 +9,16 @@
...
-        [html.Tr([html.Th(col) for col in dataframe.columns])] +
+        [html.Thead([
+            html.Tr([html.Th(col) for col in dataframe.columns])
+        ])] +
 
         # Body
```
Comment (alexcjohnson) [inline dash_docs/tutorial/examples/getting_started_table.py]:
```suggestion
```

**Comment by alexcjohnson** [comment]
Oh haha we fixed this already via a later PR - apologies for missing this one @sabas, and thanks for your contribution!

**Comment by alexcjohnson** [comment]
(#798)

**Comment by sabas** [comment]
No problem :) Thanks!

**Pull request closed without merging**